x/ibc/03-connection/keeper/verify_test.go

```diff
@@ -358,9 +409,9 @@ func (suite *KeeperTestSuite) TestVerifyPacketAcknowledgementAbsence() {
 			)
 
 			if tc.expPass {
-				suite.Require().NoError(err)
+				suite.Require().NoError(err, "valid test case: %s failed, error: %v", tc.msg, err)
```

Comment:
this is redundant because `suite.Run` is being called with `tc.msg` and the error will be printed if this fails. My preference would be not to add this